app/app/explore/page.tsx

```diff
@@ -0,0 +1,280 @@
+"use client";
+
+import { useState, useEffect } from 'react';
+import Image from 'next/image';
+import TranslucentCard from '../translucentCard';
+import { fetchJson, formatNumber } from '@/app/utils';
+
+interface Pool {
+  asset: string;
+  assetPriceUSD: string;
+  volume24h: string;
+  depth: string;
+  poolAPY: string;
+  assetDepth: string;
+  runeDepth: string;
+  nativeDecimal: string;
+}
+
+interface Saver {
+  asset: string;
+  saversCount: number;
+  saversReturn: string;
+  earned: string;
+  filled: number;
+  assetPriceUSD: string;
+  saversDepth: string;
+  assetDepth: string;
+  synthSupply: string;
+}
+
+const Explore = () => {
+  const [activeTab, setActiveTab] = useState('liquidity');
+  const [pools, setPools] = useState<Pool[]>([]);
+  const [savers, setSavers] = useState<Saver[]>([]);
+  const [runePriceUSD, setRunePriceUSD] = useState<number>(0);
+  const [loading, setLoading] = useState(true);
+  const [sortConfig, setSortConfig] = useState({ key: '', direction: '' });
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const [poolsData, statsData, saversData] = await Promise.all([
+          fetchJson('https://midgard.ninerealms.com/v2/pools'),
```

Comment:
imo we should move data fetching to a server component and pass this data into a separate client component.

Reply:
Additionally, i think we should be refreshing this data frequently once the page is rendered. stats should be periodically polled. could use tanstack query